### PR TITLE
allow the system to track you when you are online

### LINKED ISSUE #3: Allow to track your location by accepting the agreement
### Description

- create a mutation to accept that you allow to track you device whenever it is online
- register device mac address, and search for another unique identifier to use in the system

### PR DESCRIPTION
**Pull request by j4l13n**
### Description

- allow users to choose whether they need to track their location
- by allowing that they agree to send their location details

### Screenshot

...

### CODE REVIEW

File: trackme/apps/authentication/schema/mutations/user_agreement.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+from graphql import GraphQLError
+from graphql_jwt.decorators import login_required
+
+from trackme.utils.app_utils.database import get_model_object
+from ...models import User
+from trackme.utils.app_utils.validators import validator
+
+
+class Agreement(graphene.Mutation):
+    """[summary]
```
Comment (julienhirwa) [inline trackme/apps/authentication/schema/mutations/user_agreement.py]: Update the doc string message here

**Review by julienhirwa — CHANGES_REQUESTED** [review]
Good work @j4l13n, you just update the doc string needed

**Comment by j4l13n** [comment]
> Good work @j4l13n, you just update the doc string needed

Thanks, I am working on it

**Review by julienhirwa — APPROVED** [review]
Well done @j4l13n

**Pull request merged**